website/content/docs/operations/aws-oidc-provider.mdx

```diff
@@ -1,29 +1,51 @@
 ---
 layout: docs
-page_title: Nomad OpenID Connect with AWS Identity Provider
+page_title: Authenticate Nomad jobs with AWS IAM Identity Provider
 description: |-
   Integrate Nomad as an OpenID Connect (OIDC) provider with AWS IAM Identity.
 ---
 
-# Nomad OpenID Connect with AWS Identity Provider
+# Authenticate Nomad jobs with AWS IAM Identity Provider
 
-This page describes how to integrate Nomad as an OpenID Connect (OIDC) provider with AWS IAM Identity.
+This page describes how to integrate Nomad with AWS IAM Identity Provider as an OpenID Connect
+(OIDC). You can configure Nomad jobs to present a JWT token that the Nomad server authenticates 
+with AWS IAM before it executes a task.
```

Comment:
```suggestion
This page describes how to integrate Nomad with AWS IAM as an OpenID Connect (OIDC) provider.
You can configure Nomad jobs to present a JSON Web Token (JWT) that the Nomad server authenticates 
and passes to AWS IAM so that the job can access AWS services.
```